src/Bicep.LangServer/Settings/ConfigurationSettingsHandler.cs

```diff
@@ -33,13 +33,20 @@ public ConfigurationSettingsHandler(ISettingsProvider settingsProvider)
         {
             var jObject = JObject.FromObject(request);
 
-            // We currently only require - IncludeAllAccessibleAzureContainerRegistriesForCompletions settings information for module registry
+            // We currently only require - EnableModuleRegistryReferenceCompletion and IncludeAllAccessibleAzureContainerRegistriesForCompletions settings information for module registry
             // reference completion scenarios
             if (jObject["settings"] is JObject settingsObject &&
                 settingsObject["bicep"] is JObject bicepObject &&
-                bicepObject["IncludeAllAccessibleAzureContainerRegistriesForCompletions"] is JToken jToken)
+                bicepObject["experimental"] is JObject experimentalObject)
             {
-                settingsProvider.AddOrUpdateSetting(LangServerConstants.IncludeAllAccessibleAzureContainerRegistriesForCompletionsSetting, jToken.Value<bool>());
+                if (experimentalObject[LangServerConstants.IncludeAllAccessibleAzureContainerRegistriesForCompletionsSetting] is JToken includeAllAccessibleAzureContainerRegistriesForCompletionsSettingToken)
+                {
+                    settingsProvider.AddOrUpdateSetting(LangServerConstants.IncludeAllAccessibleAzureContainerRegistriesForCompletionsSetting, includeAllAccessibleAzureContainerRegistriesForCompletionsSettingToken.Value<bool>());
```

Comment:
nit: extra line?

Reply:
Hmmm I don't see extra line. Let me know if I am missing something. Thanks!

Reply:
Marking this resolved. Thanks!